calendar-app/Components/GroupSection/InGroupSection.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import { Alert, View, StyleSheet } from "react-native";
+import TextInputBar from  "../TextInputBar";
+import ButtonComp from '../ButtonComp';
+import { getUserByUsername } from '../API/Users/getUserByUsername';
+import { inviteToGroup } from '../API/Invites/InviteToGroup';
+
+const GroupInput = () => {
+  const [username, setUsername] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState(null);
```

Comment:
This user variable seems to only be used for fetchedUser, so it would be better to just remove it, becasuse it can be done without, so it would be more readable

Reply:
Loading is not used aswell